docs/metrics.md

```diff
@@ -0,0 +1,16 @@
+# Pipeline Controller Metrics
+
+Following pipeline metrics are exposed at `controller-service` on port `9090`
+
+There are different kinds of exporters available: Prometheus, Google Stackdriver, etc. They can can be configured using [observability configuration](../config/config-observability.yaml). 
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| tekton_pipelinerun_duration_seconds_[bucket, sum, count] | Histogram | `pipeline`=&lt;pipeline_name&gt; <br> `pipelinerun`=&lt;pipelinerun_name&gt; <br> `status`=&lt;status&gt; <br> `namespace`=&lt;pipelinerun-namespace&gt; | beta |
```

Comment:
What does the status "beta" means in this context?

Reply:
Well good point, I men't for more of experimental. Shall we call it out explicitly?

Thank you.

Reply:
@hrishin let's use `experimental` instead of `beta` :angel:

Reply:
@vdemeester @afrittoli amended it :)